src/Framework/App.Runtime/src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <OutDirName Condition="'$(MSBuildProjectName)' == 'Microsoft.AspNetCore.App.Runtime'">aspnetcore-sfx</OutDirName>
```

Comment:
https://github.com/dotnet/windowsdesktop/pull/4585 seems to have done it for the ref project as well. Do we need the same?

Reply:
Yes, good catch.